appservice/consumers/roomserver.go

```diff
@@ -191,6 +191,12 @@ func (s *OutputRoomEventConsumer) appserviceIsInterestedInEvent(ctx context.Cont
 		return true
 	}
 
+	if event.Type() == "m.room.member" {
```

Comment:
Please use `gomatrixserverlib.MRoomMember` as the constant

Reply:
Also, please do a nil check on `StateKey` e.g:
```
if event.Type() == gomatrixserverlib.MRoomMember && event.StateKey() != nil {
```

Reply:
Done!